.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: NPM Publish
+
+# This action will trigger on every release get created
+on:
+  release:
+    types: [created]
```

Comment:
Should this be on tag or on release?

Reply:
I was thinking release, just because it seems like a more "final" step when creating a new version